.chloggen/redisreceiver_update_redis_client_version.yaml

```
@@ -0,0 +1,29 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: redisreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Updates to newest version of redisreceiver due to security vulns in v7 (also, we should be tracking v8 anyway)
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [29600]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
+  Also helps rennovate
+  https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29502
```

```suggestion
subtext:
```

This is not needed in user-facing changelog